generated/nitclk/nitclk.proto

```diff
@@ -39,7 +32,6 @@ enum NiTClkAttribute {
   NITCLK_ATTRIBUTE_EXPORTED_SYNC_PULSE_OUTPUT_TERMINAL = 2;
   NITCLK_ATTRIBUTE_START_TRIGGER_MASTER_SESSION = 3;
   NITCLK_ATTRIBUTE_REF_TRIGGER_MASTER_SESSION = 4;
-  NITCLK_ATTRIBUTE_SCRIPT_TRIGGER_MASTER_SESSION = 5;
```

Comment:
Missing. I think in hapigen right now it's marked with 'codegen_method': 'no' in grpc_device/attributes/attributes.hapigen. You can clear that entry and this should re-generate properly.